crates/cairo-lint-core/tests/test_files/duplicate_underscore_args/duplicate_underscore_args

```diff
@@ -0,0 +1,87 @@
+//! > duplicate underscore args
```

Comment:
add a test where the first arg is the underscored and the second is the regular. Something like that:
```rs
fn foo(_test: u32, test: u32) {}
```